### PR TITLE
Medium-Publish article from markdown file

### LINKED ISSUE #2276: Medium - Publish article from Mardown file
This notebook explains how to publish an article from a Markdown file via the Medium API. It is usefull for organizations that need to quickly publish content on Medium.

### PR DESCRIPTION
**Pull request by MSaiKiran9**
## Fixes

This PR resolves #2276 

## What does this PR do?

image

It posts a Markdown file to medium as an article 

## Screenshots
Provide supporting docs, limitations if any, video of the template, or anything that you want to share to support your PR
Here I posted jupyter-naas/awesome-notebooks Readme.md as a post : 

image

### CODE REVIEW

**Comment by MSaiKiran9** [comment]
@FlorentLvr let me know if this is okay !

File: Medium/sample.ipynb [hunk 1]
```diff
@@ -0,0 +1,296 @@
+{
+ "cells": [
```
Comment (srini047) [inline Medium/sample.ipynb:2]: @MSaiKiran9 The file name has to be renamed according to the naming convention.
Have a look [here](https://github.com/jupyter-naas/awesome-notebooks/blob/master/CONTRIBUTING.md).

File: Medium/sample.ipynb [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "85e28cc8-0742-4aed-a9ee-c0b98cc00de4",
+   "metadata": {
+    "papermill": {},
+    "tags": []
+   },
+   "source": [
+    "### Setup variables\n",
+    "- `token`: Medium access token. [Instructions to get the token](https://github.com/Medium/medium-api-docs#22-authentication).\n",
```
Comment (srini047) [inline Medium/sample.ipynb:133]: image

A step-wise guide would be better. Something similar to take inspiration from, not the exact same thing.

**Review by srini047 — COMMENTED** [review]
Also, I have deleted the `sample.md` as this is not required.

**Comment by MSaiKiran9** [comment]
@srini047 I added stepwise guide in the form of comments hope that's good .

**Review by srini047 — APPROVED** [review]
LGTM🚀

**Pull request merged**